tests/data/test_btanalysis.py

```diff
@@ -108,15 +108,15 @@ def test_load_trades(default_conf, mocker):
 
 
 def test_combine_tickers_with_mean(testdatadir):
-    pairs = ["ETH/BTC", "XLM/BTC"]
+    pairs = ["ETH/BTC", "ADA/BTC"]
```

Comment:
Why all "XLM/BTC" changed to "ADA/BTC"?

Reply:
Because i thought i could remove XLM/BTC data too- but that would have an impact to backtesting running in CI so i had to revert that (it would not fail the test - but cause different results ... ).

This will allow us to remove this in the future eventually (but i kept the files in the folder for now).

I can revert this here too - but realistically, the pair does not matter for these tests - so standardizing on as few as possible seems to make sense (ADA/BTC is used surprisingly often in tests)